website/docs/getting-started/setup/installation-guides/docker/README.mdx

```diff
@@ -19,7 +19,7 @@ Docker is an open source [containerization](https://www.ibm.com/in-en/cloud/lear
 1. Create an installation folder called `appsmith` for deployment and data storage.
 2. `cd` into the installation folder.
 
-## Setup with Docker Compose _(recommended)_
+## Setup with Docker compose _(recommended)_
```

Comment:
Please keep it as before. Docker Compose is the name of the tool

Reply:
done